docs/Users_Guide/appendixB.rst

```diff
@@ -82,6 +84,10 @@ For a Gaussian grid, the syntax is
 
 The parameters **Nx** and **Ny** are as before, while **lon_zero** defines the first longitude.
 
+For a Semi Lat/Lon grid, no grid specification string is supported. This grid type is only supported via Python embedding or when reading NetCDF files generated by another MET tool. A Semi Lat/Lon grid defines the information about 2D field of data whose dimension are defined by arrays of latitude (**lats**), longitude (**lons**), level (**levels**), and time (**times**). Times are defined as unixtime, the number of seconds since January 1, 1970. Typically, the lats or lons array and the levels or times array has non-zero length. For example, a zonal mean field is defined using the lats and levels array. A meridional mean field is defined using the lons and levels array. A Hovmoeller field is defined using lats or lons versus times. A arbitrary cross-section is defined by specifying both the lats and lons array with exactly the same length versus levels or times.
+
+Statistics can be computed from data on Semi Lat/Lon grids but only when all data resides on the same Semi Lat/Lon grid. Two Semi Lat/Lon grids are equal when their lats, lons, levels, and times arrays match. No functionality is provided to regrid Semi Lat/Lon data. The MET tools can plot Semi Lat/Lon data, however no map data is overlaid since these grids lack two spatial dimensions.
```

Comment:
I think I know the answer, but just clarifying: when it's stated "Two Semi Lat/Lon grids are equal when their lats, lons, levels, and times arrays match.", that means the values of the arrays match, correct? Didn't want the confusion of "equal" meaning lengths of the arrays match.

Reply:
That's correct, the code checks that the arrays have the same length and all the array values match.